pymesync.py

```diff
@@ -42,7 +44,38 @@ def send_time(self, parameter_dict):
             response = requests.post(url, json=json_content)
             return response
         except requests.exceptions.RequestException as e:
-            # Unknown request error
+            # Request error
+            return e
+
+    def get_times(self, **queries):
+        """
+        get_times([queries])
+
+        Returns JSON times objects filtered by supplied parameters
+        """
+        query_str = ""  # Remains empty if no queries passed
+        if queries is not None:
+            # Sort them into an alphabetized list for easier testing
+            sorted_qs = sorted(queries.items(), key=operator.itemgetter(0))
+            for query, param in sorted_qs:
+                if query in self.valid_get_queries:
+                    for slug in param:
+                        query_str += "?{0}={1}".format(query, slug)
```

Comment:
This will construct the URI "times?user=example?project=aproject?start=2015-10-10", but it should actually look more like "times?user=example&project=aproject&start=2015-10-10"